mobile/README.md

```diff
@@ -0,0 +1,17 @@
+# Mobile
+
+Mobile makes use of [Go Mobile](https://pkg.go.dev/golang.org/x/mobile/cmd/gomobile). Bindings are maintained in this
+package. Mage targets are exposed to generate bindings for iOS and Android. At present, files must not be nested in
+subdirectories for bindings to be generated correctly.
+
+## iOS
+
+```
+mage ios
+```
+
+## Android
+
+```
```

Comment:
I think it's important to cover:
* Documentation on how to include this in an android/ios project, with a simple example. 
* CI that exercises actual Android/ios code so we can be confident this works.
* Guidance to readers that tells them how to discover available functionality within the SDK. 

Of course, that's a lot of scope for this PR, so tracking in separate issues is cool. But I think this is critically important.

Reply:
https://github.com/TBD54566975/ssi-sdk/issues/459